src/rime/common.h

```diff
@@ -48,7 +48,7 @@ using std::string;
 using std::vector;
 
 template <class Key, class T>
-using hash_map = std::unordered_map<Key, T>;
+using hash_map = boost::unordered_map<Key, T>;
 template <class T>
 using hash_set = std::unordered_set<T>;
```

Comment:
What about `unordered_set`? Will this get the same performance improvement?

Let alone performance, I'd be very frustrated to learn that `hash_map` and `hash_set` come from different libraries.

Reply:
> What about `unordered_set`? Will this get the same performance improvement?

Actually, `unordered_set` is used relatively few so that there is not significant improvement in my test, but I believe it can also get benefit. However, following the principle of minimum change, I left it alone.